### PR TITLE
Feature/merge release 3.0

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

File: cdap-ui/app/features/admin/controllers/NamespaceDatasetMetadataController.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 angular.module(PKG.name + '.feature.admin').controller('AdminNamespaceDatasetMetadataController',
+<<<<<<< HEAD
```
Comment (anwar6953) [inline cdap-ui/app/features/admin/controllers/NamespaceDatasetMetadataController.js]: err...

Comment (sreevatsanraman) [inline cdap-ui/app/features/admin/controllers/NamespaceDatasetMetadataController.js]: Conflict needs to be resolved

File: cdap-docs/_common/common-build.sh [hunk 2]
```diff
@@ -175,7 +175,8 @@ function build_javadocs_full() {
 function build_javadocs_api() {
   cd ${PROJECT_PATH}
   set_mvn_environment
-  MAVEN_OPTS="-Xmx512m" mvn clean package javadoc:javadoc -pl $API -am -DskipTests -P release
+#   MAVEN_OPTS="-Xmx512m" mvn clean package javadoc:javadoc -pl $API -am -DskipTests -P release
```
Comment (anwar6953) [inline cdap-docs/_common/common-build.sh]: we want to leave comment in here?

File: cdap-ui/app/features/admin/controllers/NamespaceDatasetMetadataController.js (hunk 1 above)
Comment (chtyim) [inline cdap-ui/app/features/admin/controllers/NamespaceDatasetMetadataController.js]: this is strange... Let me check one more time. Those should be done by "git checkout --theirs .."

Comment (gokulavasan) [inline cdap-ui/app/features/admin/controllers/NamespaceDatasetMetadataController.js]: This seems like a conflict

**Comment by chtyim** [comment]
Updated.

File: cdap-docs/_common/common-build.sh (hunk 2 above)
Comment (chtyim) [inline cdap-docs/_common/common-build.sh]: It's in release. @awholegunch @sreevatsanraman Can you confirm?

File: cdap-ui/app/features/admin/controllers/NamespaceDatasetMetadataController.js (hunk 1 above)
Comment (chtyim) [inline cdap-ui/app/features/admin/controllers/NamespaceDatasetMetadataController.js]: The previous merge got some issue. Probably due to a pull again. Anyway, it's fixed.

**Comment by gokulavasan** [comment]
LGTM :+1:

**Comment by chtyim** [comment]
Build passed http://builds.cask.co/browse/CDAP-DUT2006-3

**Pull request merged**